### ISSUE TITLE
"Try in Playground" fails for SPIManifest 0.13.0

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#2162]
Works for 0.11.0. Running in Arena directly shows:

```
✦ ❯ arena https://github.com/SwiftPackageIndex/SPIManifest
➡️  Package: https://github.com/SwiftPackageIndex/SPIManifest @ from(0.13.0)
🔧 Resolving package dependencies ...
Error: dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "products", intValue: nil), _JSONKey(stringValue: "Index 2", intValue: 2), CodingKeys(stringValue: "type", intValue: nil)], debugDescription: "none of the required keys found", underlyingError: nil))
```

I suspect the problem is the plugin I added in 0.12.0.

It also fails silently in "SPI Playgrounds" which isn't ideal.

**Comment by finestructure** [comment]
Make a new release after https://github.com/finestructure/Arena/issues/87 is fixed.

**Comment by finestructure** [comment]
Fixed in release 1.1.0 https://github.com/SwiftPackageIndex/SPI-Playgrounds-Releases/pull/13
